docs/schema/crawledproperties-element-searchschemaconfigurationsettings-complextypesps15xsds.md

```diff
@@ -16,11 +16,11 @@ description: "Last modified: March 09, 2015"
   
 ## Element information
 
-|||
-|:-----|:-----|
-|**Element type** <br/> |q10:CrawledPropertyInfoCollection  <br/> |
-|**Namespace** <br/> |http://schemas.datacontract.org/2004/07/Microsoft.Office.Server.Search.Portability  <br/> |
-|**Schema file** <br/> |schema_Microsoft.Office.Server.Search.Portability.xsd  <br/> |
+**Element type**: q10:CrawledPropertyInfoCollection
```

Comment:
We've been directed to keep Markdown tables as tables. You can use "Information/Location" or "Name/Value" as headers, or if you find a more specific header that fits, use that.